sklearn2pmml/decoration/__init__.py

```diff
@@ -49,4 +49,11 @@ def __init__(self, **kwargs):
 	def fit(self, X, y = None):
 		self.data_min_ = numpy.nanmin(X, axis = 0)
 		self.data_max_ = numpy.nanmax(X, axis = 0)
+		self.data_mean_ = numpy.nanmean(X, axis = 0)
+		self.data_std_ = numpy.nanstd(X, axis = 0)
+		self.data_median_ = numpy.nanmedian(X, axis = 0)
+		self.data_inter_quartile_range_ = numpy.nanpercentile(X, 75, axis = 0) - numpy.nanpercentile(X, 25, axis = 0)
+		self.data_total_freq_ = numpy.empty(X.shape[1])
+		self.data_total_freq_.fill(X.shape[0])
+		self.data_missing_freq_ = numpy.array(numpy.sum(numpy.isnan(X).astype(int), axis=0))
```

Comment:
L56 - L58 are bit ugly because we want to handle both ndarray and pandas dataframe / series...